examples/pitr/main.tf

```diff
@@ -28,5 +28,5 @@ module "postgresql_db_pitr" {
   members           = var.members
   pg_version        = var.pg_version
   pitr_id           = var.pitr_id
-  pitr_time         = data.ibm_database_point_in_time_recovery.database_pitr.earliest_point_in_time_recovery_time
+  pitr_time         = var.pitr_time == "" ? data.ibm_database_point_in_time_recovery.database_pitr.earliest_point_in_time_recovery_time : var.pitr_time
```

Comment:
why would `var.pitr_time` ever be set to an empty string ?

Reply:
If a user does not want to provide the pitr_time, it will take the earliest_point_in_time_recovery_time as PITR time.

Reply:
you need to then set the `pitr_time` variable with a default value of `null` and then update the logic here to `var.pitr_time == null`

Reply:
Changes done.